src/main/webapp/app/entities/submission.model.ts

```diff
@@ -30,6 +30,8 @@ export abstract class Submission implements BaseEntity {
     public durationInMinutes?: number;
 
     public results?: Result[];
+    public latestResult?: Result;
```

Comment:
please add documentation what this is about and move it below into a new section 

`// Helper Attributes`

Reply:
make sure to mention that it is undefined except the method below is invoked or it was explicitly set

Reply:
I added documentation and change latestResult to Result | undefined